recipes/qarchive/all/conanfile.py

```diff
@@ -104,7 +104,8 @@ def package_info(self):
         self.cpp_info.set_property("cmake_target_name", "QArchive")
         self.cpp_info.set_property("pkg_config_name", "QArchive")
         self.cpp_info.libs = ["QArchive"]
-        self.cpp_info.includedirs = [os.path.join("include", "QArchive")]
+        self.cpp_info.includedirs.append(os.path.join("include", "QArchive"))
+        self.cpp_info.requires = ["qt::qtCore", "libarchive::libarchive"]
```

Comment:
Why do we need to explicitly list the requires here?

Reply:
Because without that all qt components will be linked into final .so file and this will mess up `ldd` output (all Qt .so files will be mentioned there as dependencies for QArchive)